setup.py

```diff
@@ -3,13 +3,13 @@
 from setuptools import setup
 
 
-setup( name='algorithm'
+setup( name='state_chain.py'
```

Comment:
Why this extra `.py` in the PyPI package name?

Reply:
No reason, removed in 4dc1054fb906b8efd97b89fe04fb1bb199f57cdd.